### PR TITLE
Improve str_pad documentation

### PR DESCRIPTION
**Pull request by houssemz**
IIMHO, we need to specify that `length` is the desired length of the final padded chain. In my case, it was unclear.

### CODE REVIEW

File: reference/strings/functions/str-pad.xml [hunk 1]
```diff
@@ -42,6 +42,7 @@
      <term><parameter>length</parameter></term>
      <listitem>
       <para>
+       This is the desired length of the final padded string.
```
Comment (kamil-tekiela) [inline reference/strings/functions/str-pad.xml:45]:
```suggestion
       The desired length of the final padded string.
```

**Review by kamil-tekiela — CHANGES_REQUESTED** [review]
Right, but this is obvious, isn't it? What else would it be?

**Comment by houssemz** [comment]
> 

Thank you for the review.
Not so obvious, IMHO :)

**Comment by kamil-tekiela** [comment]
Thanks.

**Pull request merged**